business_requirement/models/business.py

```diff
@@ -269,7 +270,8 @@ def name_get(self):
         result = []
         for br in self:
             if br.ref:
-                formatted_name = u'[{}] {}'.format(br.ref, br.description)
+                formatted_name = u'[{}][{}] {}'.format(br.ref, br.name,
```

Comment:
@darshan-serpent What is the point of have twice the br.name? Perhaps you forgot add other field?